client/state/account-recovery/settings/reducer.js

```diff
@@ -1,10 +1,7 @@
-/** @format */
-
 /**
  * Internal dependencies
  */
-
-import { combineReducers, createReducer } from 'state/utils';
+import { combineReducers } from 'state/utils';
```

Comment:
Was it necessary to include these changes? I'd prefer to deprecate in one PR, then handle the migration in subsequent PRs. It would be nice to get a tracking issue as well so we can tick things off as we make incremental progress.

Reply:
I'm more than happy to split these out, of course!

My general approach in these kinds of situations (preparing for an upcoming large migration) is to handle the upfront work (in this case the deprecation) together with an example of what migrating a piece looks like. This serves as both an example, to reviewers, and as a piece of code I can point anyone else too, if they're later migrating something.

That's what I did here; I chose one part of the state tree that had tests and used `createReducer`, and another part of the state tree that also had tests and used `createReducerWithValidation`, and migrated them both.

If you're happy with how the resulting code looks like, I can easily split this up into multiple PRs, one for the deprecation, and another one (or two) for that example migration work.

Reply:
The changes in `client/state/account-recovery/reducer.js` seem very simple and happy to see those go in, but I'd prefer these to be separated if only due to of the number of lines. Additionally, account-recovery sounds like a part of the application that merits thorough smoke testing 😮

Reply:
Removed the changes from this PR. Created #35945 and #35946 with them.